tools/dataProcessor.js

```diff
@@ -121,6 +121,15 @@ function parseCashbacks() {
         cashback.img_src_url = 'https://card.discover.com' + imgTag.attr('src');
         cashback.site_name = imgTag.attr('title');
 
+        // Removing unnecessary cashback suffixes
+        let suffix_check = ['.com',' - Special',' - Featured','®','™' ]
+
+        for(let i = 0; i < suffix_check.length; i++){
+          if (cashback.site_name .indexOf(suffix_check[i]) != -1 && cashback.site_name != 'Hotels.com') {
```

Comment:
Good idea!  It will be much easier to add names this way if discover uses a company whose name includes a suffix from that list.

Would you say that this process will run quicker too because less 'stuff' is going inside the if statement?  Or is it negligible?